pkg/minikube/notify/notify_test.go

```diff
@@ -146,7 +146,7 @@ func TestMaybePrintUpdateText(t *testing.T) {
 	// test that no update text is printed if the latest version is lower/equal to the current version
 	latestVersionFromURL := "0.0.0-dev"
 	handler := &URLHandlerCorrect{
-		releases: []release{{Name: version.VersionPrefix + latestVersionFromURL}},
+		releases: []Release{{Name: version.VersionPrefix + latestVersionFromURL}},
 	}
 	server := httptest.NewServer(handler)
```

Comment:
Not a change in the PR but we should fix: I think you need to call close on httptest server

https://golang.org/src/net/http/httptest/server.go?s=2049:2093#L70